src/components/ClearingsUploader/ClearingsUploader.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import Select from '../Select/Select';
+import SelectItem from '../Select/SelectItem';
+import FileUpload from '../FileUpload/FileUpload';
+import newTheme from '../../themes/new';
+
+const EMPTY = 'EMPTY';
+const UPLOADING = 'UPLOADING';
+const COMPLETED = 'COMPLETED';
+const UPLOAD_ERROR = 'UPLOAD_ERROR';
+const FILE_TYPE_ERROR = 'FILE_TYPE_ERROR';
+
+const propTypes = {
+  text: PropTypes.object,
+  options: PropTypes.array,
+  disabled: PropTypes.bool,
+  onDropAccepted: PropTypes.func,
+  onDropRejected: PropTypes.func,
+  onRetry: PropTypes.func,
+  progress: PropTypes.number,
+  status: PropTypes.oneOf([COMPLETED, EMPTY, UPLOADING, UPLOAD_ERROR, FILE_TYPE_ERROR])
+};
+
+const defaultProps = {
```

Comment:
Remove all placeholder defaults please (all but `disabled`), mark all required props as required with `isRequired` in propTypes.
If you include defaults for messages you help users to avoid i18n, which is something we don't want :)